sale_product_set/wizard/product_set_add.xml

```diff
@@ -31,7 +31,7 @@
                             <field
                                 name="product_id"
                                 options="{'no_open': True, 'no_create': True}"
-                                attrs="{'required': [('display_type', '=', False)]}"
+                                required="display_type == 'False'"
```

Comment:
Shouldn't this be?
`required="not display_type"`

Reply:
yep...